client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -77,6 +78,36 @@ class ThemeEnhancements extends Component {
 	}
 
 	renderInfiniteScrollSettings() {
+		if ( this.props.jetpackSettingsUI ) {
+			return this.renderJetpackInfiniteScrollSettings();
+		}
+		return this.renderSimpleSiteInfiniteScrollSettings();
+	}
+
+	renderSimpleSiteInfiniteScrollSettings() {
+		const { translate } = this.props;
+		return (
+			<FormFieldset>
+				<FormLegend>{ translate( 'Infinite Scroll' ) }</FormLegend>
+
+				<div className="theme-enhancements__info-link-container site-settings__info-link-container">
+					<InfoPopover position="left">
+						<ExternalLink href="https://support.wordpress.com/infinite-scroll/" icon>
+							{ translate( 'Learn more about Infinite Scroll.' ) }
+						</ExternalLink>
+					</InfoPopover>
+				</div>
+
+				{ this.renderToggle(
+					'infinite_scroll',
+					false,
+					translate( 'Load posts as you scroll. Disable to show a clickable button to load posts' )
```

Comment:
I'm open to suggestions on the copy here.  The wp-admin copy is currently:
"Check to load posts as you scroll. Uncheck to show clickable button to load posts"

Reply:
I like the new copy here. It's more clear that one option is the default.